src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -134,7 +137,7 @@ def create(data: pd.DataFrame) -> GroupSampleData:
                     plate=row[str, "WellPlateName"],
                     entries=[
                         GroupDataElementEntry(
-                            name=column_name, value=row[float, column_name]
+                            name=column_name, value=row.get(float, column_name, NaN)
```

Comment:
Would this not lead to other non `Masked` `Range?` strings being cast to NaN maybe Im missing something in the logic upstream though!

Reply:
@rayaankha-tara your reading here is correct. 

In this case, I think it's acceptable to treat all invalid values as NaN

Since this is machine generated data, we typically don't have to be as worried about weird unintentional cases.

I think this will be further improved when the error document is added, when we can put the value that caused the nan in the error document.